include/device.h

```diff
@@ -337,6 +337,38 @@ size_t z_device_get_all_static(const struct device * *devices);
  */
 bool z_device_ready(const struct device *dev);
 
+/** @brief Determine whether a device is ready for use
+ *
+ * This is the implementation underlying `device_usable_check()`, without the
+ * overhead of a syscall wrapper.
+ *
+ * @param dev pointer to the device in question.
+ *
+ * @return true if and only if the device is available for use.
```

Comment:
This is wrong; I assume you meant the semantics documented in device_usable_check, right?

Reply:
Yes, fixed, thanks.